src/Root.cpp

```diff
@@ -558,6 +558,14 @@ void cRoot::SaveAllChunks(void)
 
 
 
+void cRoot::SendPlayerLists(cPlayer * a_DestPlayer)
+{
+	for (WorldMap::iterator itr = m_WorldsByName.begin(), end = m_WorldsByName.end(); itr != end; ++itr)
```

Comment:
A `WorldMap::iterator` is, but `world` in `for (const auto & world: m_WorldsByName)` isn't - it is a `std::pair<AString, cWorld *>`.
Ref.: http://stackoverflow.com/questions/6963894/c11-how-to-use-range-based-for-loop-with-stdmap
For this reason I think it's not worth the change.